client/components/repository/common/Sidebar/Tasks/CreateDialog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <tailor-dialog
+    v-model="visible"
+    header-icon="mdi-folder-plus-outline">
+    <template #activator="{ on }">
+      <v-btn v-on="on" text small>
+        <v-icon small>mdi-plus</v-icon>
+        Create task
+      </v-btn>
+    </template>
+    <template #header>
+      <h4 class="h4">Add task</h4>
+    </template>
+    <template #body>
+      <validation-observer
+        ref="form"
+        @submit.prevent="$refs.form.handleSubmit(create)"
+        tag="form">
+        <task-field-group v-bind.sync="task" />
+        <div class="d-flex justify-end">
+          <v-btn @click="visible = false" text>Cancel</v-btn>
+          <v-btn type="submit" color="primary" text>Create</v-btn>
+        </div>
+      </validation-observer>
+    </template>
+  </tailor-dialog>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+import { priorities } from 'shared/workflow';
+import TailorDialog from '@/components/common/TailorDialog';
+import TaskFieldGroup from '@/components/repository/WorkflowBoard/Sidebar/FieldGroup';
+
+const initTaskState = statuses => ({
+  description: null,
+  status: statuses ? statuses[0].id : null,
+  assigneeId: null,
+  priority: priorities[2].id,
```

Comment:
Don't like this 2 here 😬, can we filter based on something and default to null (if it changes over time, so there is a fallback / it doesn't break)